### DISCUSSION TITLE
Supported Ubuntu Version

### DISCUSSION THREAD

**Discussion by HuntTheSun** [Ylianst/MeshCentral#3624] [General]
Hello everyone,

I hope this is the right channel for this type of stuff, I am a bit new to GitHubs features.
I just wanted to say that I installed MeshCentral on Ubuntu 20.04 (the 18.04 had a too near EOL for my taste) and after testing it I ran into no issues. I configured it exactly like the manual told me (with MongoDB) and it works like a charm.
So my question is if you could add Ubuntu 20.04 to the supported distros (if you find the time to validate functionality)?

By the way, I am blown away at the features, this is exactly what I needed and more.

**Comment by JSuenram** [comment]
Hi and welcome!
As long as there are the required packages available, Ubuntu as most other distributions should work.
NodeJS, NPM and then most if not all needed Software is installed by MeshCentral.js when started first time from command line.

We are also using several MC-Setups on Ubuntu 20.04 and Debian 11.

So I think you are good to go with Ubuntu....

**Comment by JSuenram** [comment]
@HuntTheSun 
If your question is answered pls mark as answered.
Thank you!

**Reply by HuntTheSun** [reply]
> @HuntTheSun If your question is answered pls mark as answered. Thank you!

Sorry, I dont think I can mark it as answered since I did not create it in the Q&A section.
If there is another way to close this thread I cant seem to find it.
Thank you for your answer

Also, thank you Ylianst for your detailed answer.

**Comment by Ylianst** [comment]
In order to support as many Linux distros as possible, the MeshAgent for Linux is compiled on a very old machine with Ubuntu 16.10. Any Linux after that should work. Bryan also added support for initd, the service manager prior to systemd and Bryan has 20+ Linux distros he does testing on. So, it should work on almost Linux.

One issue is that the Window manager may change from distro to distro and that will cause issues. Bryan is always trying to fix things, but it's difficult work.

**Reply by krayon007** [reply]
> In order to support as many Linux distros as possible, the MeshAgent for Linux is compiled on a very old machine with Ubuntu 16.10. 

It's actually built on Ubuntu 10.04

>Any Linux after that should work. Bryan also added support for initd, the service manager prior to systemd and Bryan has 20+ Linux distros he does testing on. So, it should work on almost Linux.

I have 50 different Linux VMs that I test with. If you include other OSes, I have over 80 VMs. I test on as many different distros, configured with several different window managers, etc. Gnome, KDE, Cinnamon, XFCE, etc.
